Snabble/UI/Scanner/ShoppingListCellView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  ShoppingListCellView.swift
+//  Snabble
+//
+//  Created by Anastasia Mishur on 7.04.22.
+//
+
+import Foundation
+
+final class ShoppingListCellView: UIView {
+    private weak var productImage: UIImageView?
+    public weak var spinner: UIActivityIndicatorView?
+
+    private weak var nameLabel: UILabel?
+
+    private weak var quantityLabel: UILabel?
+    private weak var checkContainer: UIView?
+    private weak var checkImage: UIImageView?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setupUI()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupUI() {
+        let imageViewLayuotGuide = UILayoutGuide()
+
+        let productImage = UIImageView()
+        productImage.translatesAutoresizingMaskIntoConstraints = false
+        productImage.contentMode = .scaleAspectFit
+        productImage.tintColor = .label
+
+        let spinner = UIActivityIndicatorView()
+        spinner.translatesAutoresizingMaskIntoConstraints = false
+        spinner.hidesWhenStopped = true
+        if #available(iOS 13.0, *) {
+            spinner.style = .medium
+        } else {
+            spinner.style = .gray
+        }
+        spinner.contentMode = .scaleToFill
```

Comment:
Is this needed? I don't see a reason for it. Why did you add it?

Reply:
Adding `spinner.style = .medium` gives an error _"medium' is only available in iOS 13.0 or newer"_, as SDK has current deployment target iOS version 12.4. Otherwise, using only `spinner.style = .gray` is deprecated for iOS 13.0+ and it's recommended to use `.medium` instead. So while SDK iOS version is lower than 13.0, we should define both cases of styling.

Reply:
I just mean line 45 `spinner.contentMode = .scaleToFill`

Reply:
I'm sorry for the misunderstanding, removed content mode case, as it's not needed.